content/en/monitors/notify/_index.md

```diff
@@ -42,67 +42,6 @@ Steps to free up disk space:
 4. Remove duplicate files
 ```
 
-### Tags
-
-Add tags to your monitor. Monitor tags are different from metric tags. They are used in the UI to group and search for monitors. If tag policies are configured, the required tags and tag values need to be added. To learn more, see [Tag Policies][6].
-
-{{< img src="monitors/notifications/notifications_add_required_tags.png" alt="View of policy tag configuration. Underneath 'Policy tags' are three example tags, cost_center, product_id, and env, next to a 'Select value' dropdown." style="width:100%;" >}}
-
-### Renotify
-
-Enable monitor renotification (optional) to remind your team that a problem is not solved.
-
-  {{< img src="monitors/notifications/renotify_options.png" alt="Enable renotify" style="width:90%;" >}}
-
-Configure the renotify interval, the monitor states from which the monitor renotifies (within `alert`, `no data`, and `warn`) and optionally set a limit to the number of renotification messages sent.
-
-For example, configure the monitor to `stop renotifying after 1 occurrence` to receive a single escalation message after the main alert.
-**Note:** [Attribute and tag variables][7] in the renotification are populated with the data available to the monitor during the time period of the renotification.
-
-If renotification is enabled, you are given the option to include an escalation message that is sent if the monitor remains in one of the chosen states for the specified time period.
-
-
-The escalation message can be added in the following ways:
-
-* In the `{{#is_renotify}}` block in the original notification message (recommended).
-* In the *Renotification message* field in the `Notify your team` section.
-* With the `escalation_message` attribute in the API.
-
-If you use the `{{#is_renotify}}` block, the original notification message is also included in the renotification, so:
-
-1. Include only extra details in the `{{#is_renotify}}` block and don't repeat the original message details.
-2. Send the escalation message to a subset of groups.
-
-Learn how to configure your monitors for those use cases in the [example section][8].
-
-### Priority
-
-Add a priority (optional) associated with your monitors. Values range from P1 through P5, with P1 being the highest priority and the P5 being the lowest.
-To override the monitor priority in the notification message, use `{{override_priority 'Pi'}}` where `Pi` is between P1 and P5.
-
-For example, you can set different priorities for `alert` and `warning` notifications:
-
-```
-{{#is_alert}}
-{{override_priority 'P1'}}
- ...
-{{/is_alert}}
-
-{{#is_warning}}
-{{override_priority 'P4'}}
-...
-{{/is_warning}}
-```
-
-## Notify your team
-
-Use the **Notify your team** section to:
```

Comment:
Could this go in the ##Notify your team section at the top of the page?

Reply:
Indeed, good catch!